distributed-exceptions/src/test/java/io/axoniq/distributedexceptions/DistributedExceptionsApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package io.axoniq.distributedexceptions;
```

Comment:
I'd wager we could drop this guy altogether.

Reply:
We could, but it's also nice to at least have verification that the tests are run and that the application can startup. In case there are version changes in the futures. Granted, it was using both profiles in the same application, I split those into two different test classes

Reply:
However, since the configuration requires AxonServer to be up and running, and the goal of this example is not how to create integration tests against AxonServer, I removed both of these tests